Tests/EventListener/ParamFetcherListenerTest.php

```diff
@@ -79,6 +74,7 @@ public function testSettingParamFetcherOnRequest()
 
     public function testParamFetcherOnRequestNoZone()
     {
+        $this->markTestSkipped();
```

Comment:
I am wondering how the test passed before as the event does not return a callable controller.